ui/pages/confirmations/components/confirm/confirm-alerts/confirm-alerts.tsx

```diff
@@ -3,13 +3,23 @@ import { AlertActionHandlerProvider } from '../../../../../components/app/alert-
 import useConfirmationAlertActions from '../../../hooks/useConfirmationAlertActions';
 import setConfirmationAlerts from '../../../hooks/setConfirmationAlerts';
 import { AlertMetricsProvider } from '../../../../../components/app/alert-system/contexts/alertMetricsContext';
+import { useConfirmationAlertMetrics } from '../../../hooks/useConfirmationAlertMetrics';
 
 const ConfirmAlerts = ({ children }: { children: ReactElement }) => {
+  const { trackAlertActionClicked, trackAlertRender, trackInlineAlertClicked } =
+    useConfirmationAlertMetrics();
+
   const processAction = useConfirmationAlertActions();
   setConfirmationAlerts();
 
   return (
-    <AlertMetricsProvider>
+    <AlertMetricsProvider
+      metrics={{
```

Comment:
Great spot, there could be alternate systems in future using the alert metrics that won't want to add metrics to transactions and signatures.